src/CodiceFiscale/Calculator.php

```diff
@@ -49,7 +51,7 @@ public function calculate()
     /**
      * Calculate all possibilities for the code fiscale.
      *
-     * @returns Returns the complete codice fiscale.
+     * @return array Returns the complete codice fiscale.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.